tutorial/source/air.ipynb

```diff
@@ -150,16 +139,14 @@
     "def prior_step_sketch(t):\n",
     "    # Sample object pose. This is a 3-dimensional vector representing x,y position and size.\n",
     "    z_where = pyro.sample('z_where_{}'.format(t),\n",
-    "                          dist.normal,\n",
-    "                          z_where_prior_mu.expand(1, -1),\n",
-    "                          z_where_prior_sigma.expand(1, -1),\n",
+    "                          dist.Normal(z_where_prior_mu.expand(1, -1),\n",
+    "                                      z_where_prior_sigma.expand(1, -1)),\n",
     "                          extra_event_dims=1)\n",
     "\n",
     "    # Sample object code. This is a 50-dimensional vector.\n",
     "    z_what = pyro.sample('z_what_{}'.format(t),\n",
-    "                         dist.normal,\n",
-    "                         z_what_prior_mu.expand(1, -1),\n",
-    "                         z_what_prior_sigma.expand(1, -1),\n",
+    "                         dist.Normal(z_what_prior_mu.expand(1, -1),\n",
+    "                                     z_what_prior_sigma.expand(1, -1)),\n",
     "                         extra_event_dims=1)\n",
```

Comment:
`extra_event_dims` is an arg of `TorchDistribution`, not of `pyro.sample`. I believe this is causing your test failures.